src/doc/trpl/const-and-static.md

```diff
@@ -0,0 +1,86 @@
+% `const` and `static`
+
+Rust has a way of defining constants with the `const` keyword:
+
+```rust
+const N: i32 = 5;
+```
+
+Unlike [`let`][let] bindings, you must annotate the type of a `const`.
+
+[let]: variable-bindings.html
+
+Constants live for the entire lifetime of a program. More specifically,
```

Comment:
I think this would read easier as just:

Constants in Rust have no fixed address in memory because they're effectively inlined to each place they are used. For this reason, references to the same constant are not guaranteed to refer to the same memory address.

Particularly just eliminating the bit about how long constants live, because it's confusing compared to the description of statics.